Makefile

```diff
@@ -33,9 +33,7 @@ frontend/start:
 .env: .env.example
 	@if [ -f .env ]; \
 	then\
-		echo '\033[1;41m/!\ The .env.example file has changed. Please check your .env file (this message will not be displayed again).\033[0m';\
-		touch .env;\
-		exit 1;\
+		echo '\033[1;41m/!\ You already have created .env file. You should have a look at the .env.example file to ensure you have set all required environment variables for docker-compose.\033[0m';\
```

Comment:
We shouldn't exit the script if there is already a file. The wording was also not defining what was the problem.